substratevm/mx.substratevm/mx_substratevm.py

```diff
@@ -468,6 +468,19 @@ def svm_gate_body(args, tasks):
                 cinterfacetutorial([])
                 clinittest([])
 
+        with Task('image demos debuginfo', tasks, tags=[GraalTags.helloworld_debug]) as t:
+            if t:
+                if svm_java8():
```

Comment:
You should use `'-H:GenerateDebugInfo=1'` also for
* `javac_image(...)`
* First `helloworld(...)` build (non-shared library build of hello world image)
* `cinterfacetutorial(...)`
Otherwise it's pointless to build those images in `GraalTags.helloworld_debug`

Reply:
Ah ok. I had no idea what that script was doing. I hope I have got it right this time.

Reply:
I also fixed the eclipse format problems.

Reply:
The gate seems to be blocked however and closing then re-opening the PR seems not to have restarted testing.

Reply:
That's annoying. Sorry to hear. You should be able to run your new gate task locally with
```
mx gate --omit-clean --tags build,helloworld_debug
```

Reply:
Btw, you forgot to also add `helloworld_debug` to https://github.com/oracle/graal/blob/51c8f8f55d012aa5f073a608de6b66931c419c00/substratevm/mx.substratevm/mx_substratevm.py#L361

Reply:
... and here: https://github.com/oracle/graal/blob/97876cd0075b1cdd169342af8125e2a3132ec47e/.travis.yml#L29

Reply:
Ok, I added those changes (although I have no proper idea what they are meant to achieve :-) and the above mx gate command appears to be do its stuff without errors. Also, the Travis gate build seems to be running again after pushing those changes.